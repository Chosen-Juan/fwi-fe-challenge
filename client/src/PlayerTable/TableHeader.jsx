import React from 'react';
import Sort from './Sort/Sort';

const TableHeader = () => (
  <thead className="table-head">
    <tr>
      <th />
      <th>
        Player
        <Sort playerKey="name" />
      </th>
      <th>
        Winnings
        <Sort playerKey="winnings" />
      </th>
      <th>
        Native of
        <Sort playerKey="country" />
      </th>
      <th className="is-hidden-mobile">Controls</th>
    </tr>
  </thead>
);

export default TableHeader;
