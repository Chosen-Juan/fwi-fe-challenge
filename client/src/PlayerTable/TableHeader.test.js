import React from 'react';
import { shallow } from 'enzyme';

import TableHeader from './TableHeader';

describe('TableHeader', () => {
  const tableHeader = shallow(<TableHeader />);

  it('should match snapshot', () => {
    expect(tableHeader).toMatchSnapshot();
  });

});
