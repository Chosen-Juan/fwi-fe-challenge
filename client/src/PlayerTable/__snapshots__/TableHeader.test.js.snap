// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader should match snapshot 1`] = `
<thead
  className="table-head"
>
  <tr>
    <th />
    <th>
      Player
      <Connect(Sort)
        playerKey="name"
      />
    </th>
    <th>
      Winnings
      <Connect(Sort)
        playerKey="winnings"
      />
    </th>
    <th>
      Native of
      <Connect(Sort)
        playerKey="country"
      />
    </th>
    <th
      className="is-hidden-mobile"
    >
      Controls
    </th>
  </tr>
</thead>
`;
